web/client/static/RULES.md

```diff
@@ -0,0 +1,2 @@
+# Implemented notices
```

Comment:
Should this file be committed? I think it's copied from https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md during build with gradle.